practice-app/website/views.py

```diff
@@ -75,3 +76,23 @@ def create_event():
     
     return render_template("create_event.html", user= current_user, sports=sports)
 
+@views.route('show_badge/', methods=['POST','GET'])
+def show_badge():
+    if request.method == "POST":
+
+        req = "http://localhost:5000/api/v1.0/badges/show_badge/"
+        headers = {'Content-type': 'application/json'}
+        response = requests.post(req, headers=headers)
+        json_response = response.json()
+        
+        if response.status_code != 201:
+            flash('Error Occured, Try Again Later', category='error')
+
+        badges = []
+        for badge in json_response['badges']:

```

Comment:
i could not understand that part. can you explain?

Reply:
Since we can not transfer Badge objects from api to the views, we have to jsonify them. After the view gets the json files, that part converts them to the Badge objects again.

Reply:
there are too many badges. my bad, sorry